tests/e2e/tests/test_sanity.py

```diff
@@ -37,10 +39,11 @@
 from e2e.utils.load_balancer.common import CONFIG_FILE as LB_CONFIG_FILE
 from kfp_server_api.exceptions import ApiException as KFPApiException
 from kubernetes.client.exceptions import ApiException as K8sApiException
-
+from e2e.fixtures.kserve_denpendencies import  kserve_iam_service_account, kserve_secret, clone_tensorflow_serving, s3_bucket_with_training_data, kserve_inference_service
 
 INSTALLATION_PATH_FILE = "./resources/installation_config/vanilla.yaml"
 CUSTOM_RESOURCE_TEMPLATES_FOLDER = "./resources/custom-resource-templates"
+KNATIVE_CONFIG_FILE = "./resources/kserve/knative-config.yaml"
```

Comment:
this file doesn't exist in pr under resources/kserve?

Reply:
it's generated in this step during the test. https://github.com/awslabs/kubeflow-manifests/blob/299b3a8c874f213668f8ea1fe1553f9eed2fca50/tests/e2e/tests/test_sanity.py#L251

Reply:
will add comment to show the directory is going to be generated